src/desktop/apps/artist2/server.js

```diff
@@ -44,11 +44,13 @@ app.get('/artist2/:artistID*', async (req, res, next) => {
       res.locals.sd.REFERRER &&
       !res.locals.sd.REFERRER.includes(res.locals.sd.APP_URL)
 
-    // Since this page is admin-only now, need to swap this in to test.
-    // res.locals.sd.ARTIST_PAGE_CTA_ENABLED = true
     res.locals.sd.ARTIST_PAGE_CTA_ENABLED = !user && isExternalReferer
     res.locals.sd.ARTIST_PAGE_CTA_ARTIST_ID = req.params.artistID
 
+    // While we are rolling out the new page, override the default (`artist`)
+    // type inferred from the URL, for tracking and comparison purposes.
+    res.locals.sd.PAGE_TYPE = 'new-artist'
```

Comment:
Sorta weird to call this `new-artist` but `artist2` everywhere else. Doesn't bother me much though.

Do we need a similar change in the `mobile/` tree, or is this implementation responsive and handle both?

Reply:
Yea, we called it `artist2` in terms of this directory structure, but it's 'Artist' pretty much everywhere. I can rename to anything you'd like, if it matters!

Yep, mobile requests will get this automatically.